x-pack/plugins/apm/common/anomaly_detection.ts

```diff
@@ -4,9 +4,73 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
+
 export interface ServiceAnomalyStats {
   transactionType?: string;
   anomalyScore?: number;
   actualValue?: number;
   jobId?: string;
 }
+
+export enum severity {
+  critical = 'critical',
+  major = 'major',
+  minor = 'minor',
+  warning = 'warning',
+}
+
+export function getSeverity(score?: number) {
+  if (typeof score !== 'number') {
+    return undefined;
+  } else if (score < 25) {
+    return severity.warning;
+  } else if (score >= 25 && score < 50) {
+    return severity.minor;
+  } else if (score >= 50 && score < 75) {
+    return severity.major;
+  } else if (score >= 75) {
+    return severity.critical;
+  } else {
+    return undefined;
+  }
+}
+
+// error message
```

Comment:
I like when we have comments on our code which explains difficult logic, but `MLErrorMessages` is already pretty clear, maybe you could remove it?